src/bin/CMakeLists.txt

```diff
@@ -44,7 +44,10 @@
 #  Kathleen Biagas, Thu Mar 24 16:29:50 PDT 2016
 #  Remove qt.conf in favor of having FindVisItQtx.cmake create one on the fly.
 #
-#****************************************************************************
+#  Alister Maguire, Tue Oct 15 10:09:02 PDT 2019
+#  Copy over the makemili_driver to the exe directory. 
+#
+#****************************************************************************/
```

Comment:
You can remove the "/" at the end of the line.

Reply:
Thanks!